npm_and_yarn/lib/dependabot/npm_and_yarn/file_updater/npm_lockfile_updater.rb

```diff
@@ -365,11 +365,12 @@ def dependencies_in_error_message?(error_message)
         def write_temporary_dependency_files(lockfile_name,
                                              update_package_json: true)
           write_lockfiles(lockfile_name)
-          File.write(".npmrc", npmrc_content)
```

Comment:
It should also work if we write this into the directory where we write the lockfile instead of into each package json directory (there might be hundreds of package json files in a monorepo and we only run `npm install` in the directory where we write this lockfile). Something like:

```
lockfile_dir = Pathname.new(lockfile_name).dirname
npmrc_path = lockfile_dir.join(".npmrc")
File.write(npmrc_path, npmrc_content)
```